### DISCUSSION TITLE
much less WIFI Reconnects with 8.33

### DISCUSSION THREAD

**Discussion by kentbrock** [technyon/nuki_hub#464] [General]
Hi, 

this is a great project. Thanks for all the efforts :-)

I´ve tried a lot of Nuki Hub SW Versions on my ESP32 even the newest one (9.0). Via telegram (MQTT -> FHEM -> Telegram) I will be informed of the status of the lock. Unfortunately when WIFI gets reconnect after loss (or what ever reason) I get a message that the state is locked. With SW version 8.33 there are nearly no reconnects or "wrong" messages of the status of the lock. And the avaiblity of the WIFI connections seems to be perfect.

Maybe this is an observation this helps for future development.

BR

Markus

**Comment by 0asisX** [comment]
+1, I've tried all the new releases and always ended up reverting back to 8.33 due to connection instabilities

**Comment by iranl** [comment]
If you have a problem please open an issue and provide all the required information on the most recent build (should probably wait for release 9.02 which is coming soon)
